astropy/coordinates/builtin_frames/galactic_transforms.py

```diff
@@ -23,7 +23,7 @@ def fk5_to_gal(fk5coord, galframe):
     mat2 = rotation_matrix(90 - Galactic._ngp_J2000.dec.degree, 'y')
     mat3 = rotation_matrix(Galactic._ngp_J2000.ra.degree, 'z')
 
-    return mat1 * mat2 * mat3 * pmat
+    return matmul(matmul(matmul(mat1, mat2), mat3), pmat)
```

Comment:
It indeed needs to be something within a comment, since one cannot have `@` as part of the code, even if protected by an `if` statement.

I do think, though, that the code will look clean enough if I define `matrix_product` (which repeatedly multiplies matrices together) and perhaps also a `matrix_transpose`.